scripts/commit_retriever.py

```diff
@@ -18,18 +18,26 @@ def __init__(self, cache_root):
         assert os.path.isdir(cache_root), f"Cache root {cache_root} is not a dir."
         self.repo_dir = os.path.join(cache_root, "mozilla-central")
 
-    def retrieve_commits(self):
+    def retrieve_commits(self, limit):
         repository.clone(self.repo_dir)
 
-        if not db.is_old_version(repository.COMMITS_DB):
+        if not db.is_old_version(repository.COMMITS_DB) and not limit:
             db.download(repository.COMMITS_DB, support_files_too=True)
 
             for commit in repository.get_commits():
                 pass
 
             rev_start = f"children({commit['node']})"
+
+            if limit:
+                logger.warning("Limit argument was ignored, the DB was used instead")
```

Comment:
This should never happen because we have "and not limit"

Reply:
With the current code yes, do you want me to remove it or keep it so a future refactoring that might change when the limit is enforced or not would not have to add it back?

Reply:
Let's remove it